modules/standard/Socket.chpl

```diff
@@ -0,0 +1,481 @@
+/*
+ * Copyright 2020-2021 Hewlett Packard Enterprise Development LP
+ * Copyright 2004-2019 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+module Socket {
+
+  public use Sys;
+  use Time;
+  use SysCTypes;
+  use SysError;
+  use SysBasic;
+  use CPtr;
+  use IO;
+
+  enum IPFamily {
+    IPv4 = 2,
+    IPv6 = 10,
+    IPUnspec = 0
```

Comment:
I assume IPUnspec is an enum named by you, if so, I feel "IPNoSpec" would be better.

Reply:
It's a POSIX translation of `AF_UNSPEC` as we can't have `c_int` in enum I used their values directly